src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-activemq.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: JMX monitoring with ActiveMQ
+tags:
+  - Integrations
+  - On-host integrations
+  - On-host integrations list
+  - ActiveMQ dashboard
+  - ActiveMQ with New Relic
+metaDescription: 'Get a tour of our quickstarts and dashboards that take JMX-scraped data from your favorite automation software and houses it in the New Relic platform.'
+--- 
+
+import activemqLandingPage from 'images/activemqLandingPage.png'
+
+import accessDashboards from 'images/Access_dashboards.png' 
+
+With our ActiveMQ dashboard, you can easily track your ActiveMQ logs, keep an eye on your critical brokers, and get a total overview of all components of your messaging system. Built with our Linux infrastructure agent and our JMX integration, ActiveMQ dashboards take advantage of JMX scraping so you can view all your most important data, all in one place.
+
+<img
+  src={activemqLandingPage}
+  title="ActiveMQ dashboard landing page"
+  alt="A screenshot of a dashboard with ActiveMQ metrics. The dashboard ."
+/>
+<figcaption>
+After setting up ActiveMQ with New Relic, your data will display in a quickstart dashboard like this one, out of the box. 
+</figcaption> 
+
+## Set up the JMX integration [#setup]
+
+Before getting ActiveMQ data into New Relic, first install our infrastructure agent, then expose your metrics by installing the JMX integration. If you haven't already:
```

Comment:
If goal is having them install JMX integration, I think we can just point to that doc and don't have to re-explain that in redundant way. We can just say 'You'll need to install our JMX integration' and link to it. That's the general approach we should take when referencing existing docs to avoid redundancy.

Reply:
if there's something we think is lacking from those other instructions, we should try to fix that in those other install docs and avoid creating new integration install instructions. that forces us to fix any problems at the origin.

Reply:
That makes sense. Thanks!

Reply:
Was revisiting this comment. Do you think I should just eliminate this h2 and under ## Get ActiveMQ metrics in dashboard, link to the JMX install doc, so it reads something like this:

`
## Get ActiveMQ metrics in a dashboard [#send]

Our ActiveMQ dashboard takes data scraped from your JMX integration and translates it into a dashboard. Storing your data in New Relic gives you a full overview of your system's health and critical business points. If you haven't already, you'll need to install our [JMX monitoring integration](/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-integration).
`

Reply:
yeah I think that's fine